webpack/components/extensions/HostDetails/DetailsTabCards/ImageModeCard.js

```diff
@@ -33,6 +33,7 @@ const ImageModeCard = ({ isExpandedGlobal, hostDetails }) => {
       expandable
       masonryLayout
       isExpandedGlobal={isExpandedGlobal}
+      ouiaId="image-mode-card"
```

Comment:
Non-blocking, but maybe leave out "card" here? otherwise we end up with `card-template-image-mode-card` and it's a card sandwich

Reply:
mm card sandwich

Reply:
done!